packages/fluentui/react-northstar/src/themes/teams/components/Form/formFieldStyles.ts

```diff
@@ -5,20 +5,6 @@ import { FormFieldVariables } from './formFieldVariables';
 
 const formFieldStyles: ComponentSlotStylesPrepared<FormFieldStylesProps, FormFieldVariables> = {
   root: ({ props, variables }): ICSSInJSStyle => ({}),
-  label: ({ props }): ICSSInJSStyle => {
-    const { type, inline, required } = props;
-    return {
-      ...((!type || (type !== 'radio' && type !== 'checkbox')) && {
-        display: 'block',
-      }),
-      ...(inline && { marginRight: pxToRem(10), display: 'inline' }),
-      ...(required && {
-        '::after': {
-          content: '"*"',
-        },
-      }),
-    };
-  },
```

Comment:
This is a breaking change, please add it to the changelog.

Reply:
I am reverting it as we still have `FormField` and it will be needed there